src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -383,24 +454,26 @@ void GcsResourceManager::AddResourcesChangedListener(std::function<void()> liste
 
 void GcsResourceManager::UpdateNodeNormalTaskResources(
     const NodeID &node_id, const rpc::ResourcesData &heartbeat) {
-  auto iter = cluster_scheduling_resources_.find(node_id);
-  if (iter == cluster_scheduling_resources_.end()) {
-    return;
-  }
-
-  auto &scheduling_resoruces = iter->second;
-  ResourceSet resources_normal_task(MapFromProtobuf(heartbeat.resources_normal_task()));
-  if (heartbeat.resources_normal_task_changed() &&
-      heartbeat.resources_normal_task_timestamp() >
-          latest_resources_normal_task_timestamp_[node_id] &&
-      !resources_normal_task.IsEqual(scheduling_resoruces->GetNormalTaskResources())) {
-    scheduling_resoruces->SetNormalTaskResources(resources_normal_task);
-    latest_resources_normal_task_timestamp_[node_id] =
-        heartbeat.resources_normal_task_timestamp();
-    for (const auto &listener : resources_changed_listeners_) {
-      listener();
-    }
-  }
+  // TODO(Shanly): update normal task resources.
```

Comment:
Why normal task updating cannot be done in this PR?

Reply:
Because there are only `total` and `normal` fields in `NodeResources` for the time being.

Reply:
emm, I will remove normal task related code for the time being, and @Chong-Li will refactor it later.